pkgs/applications/office/beancount/beancount-ing-diba.nix

```
@@ -0,0 +1,32 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, python3Packages
+, poetry }:
+
+python3Packages.buildPythonApplication rec {
+  pname = "beancount-ing-diba";
+  version = "0.5.0";
+
+  src = fetchFromGitHub {
+    owner = "siddhantgoel";
+    repo = "beancount-ing-diba";
+    rev = "v${version}";
+    sha256 = "1wzcxkfhv2lqfvrsh7x2bslsav46ybyx96g3058j5f80kllbrsl2";
+  };
+
+  format = "pyproject";
+
+  nativeBuildInputs = [
+    poetry
+  ];
+
+  meta = {
+    homepage = "https://github.com/siddhantgoel/beancount-ing-diba";
+    description = "Beancount Importers for ING-DiBa (Germany) CSV Exports";
+    license = lib.licenses.mit;
```

```suggestion
    license = licenses.mit;
```
 same for `meta.maintainers`